README.md

```diff
@@ -1,9 +1,41 @@
-# Grouparoo `app-example-config`
+# Grouparoo Example: Declarative Sync
 
-This is an example application that is configured using Declarative Sync. [See the docs](https://www.grouparoo.com/docs/config/code-config) for more information on configuring your Grouparoo applications with code.
+This is an example Grouparoo application that is configured using Declarative Sync. It will sync data from a local SQLite database (created from CSV files) to Mailchimp.
 
-This is the code that was used in the following video: https://www.youtube.com/watch?v=kQ789gMXJB8
+You can learn more about declarative sync by [reading the documentation](https://www.grouparoo.com/docs/config/code-config). Or take a brief tour by watching [this video](https://www.youtube.com/watch?v=kQ789gMXJB8).
 
----
+[![Grouparoo Declarative Sync](https://img.youtube.com/vi/kQ789gMXJB8/0.jpg)](https://www.youtube.com/watch?v=kQ789gMXJB8)
+
+## Setup
+
+To get started, first install the dependencies:
+
+    $ npm install
+
+### Setup Source Database
+
+This project uses a SQLite source. If you don't have SQLite installed locally, [install it](https://www.tutorialspoint.com/sqlite/sqlite_installation.htm).
+
```

Comment:
I *think* you don't actually need to install SQLite, the node_module does it for you (makes a local copy of the binary)

Reply:
Confirmed - I don't actually have the sqlite binary in my path, but Grouparoo works just fine